python/lsst/pipe/tasks/multiBand.py

```diff
@@ -697,6 +697,19 @@ class MergeMeasurementsConfig(MergeSourcesConfig):
     pseudoFilterList = ListField(dtype=str, default=["sky"],
                                  doc="Names of filters which may have no associated detection\n"
                                      "(N.b. should include MergeDetectionsConfig.skyFilterName)")
+    snName = Field(dtype=str, default="base_PsfFlux",
+                       doc="Name of flux measurement for calculating the S/N when choosing the "
+                           "reference band.")
+    minSN = Field(dtype=float, default=10.,
+                  doc="If the S/N from the priority band is below this value (and the S/N "
+                      "is larger than minSNDiff compared to the priority band), use the band with "
+                      "the largest S/N as the reference band."
+                  )
```

Comment:
For consistency with the code above, please move the ``)`` onto the previous line.